### ISSUE TITLE
Easy way to add new font

### ISSUE THREAD

**Issue by namcktv** [sbrin/lopaka#83]
Is there any way to add fonts easily, such as adding them directly on the GUI, or adding them to the "fonts" folder?

**Comment by sbrin** [comment]
There’s no way. We’re working on adding more fonts. Currently we’re struggling with different bdf formats

**Comment by namcktv** [comment]
The u8g2 library has all the bdf fonts, but why can't it be added to the app?

**Comment by sbrin** [comment]
@namcktv it takes time 😀

**Comment by krbvroc1** [comment]
I dropped by to concur. I am trying to design a display and was looking for a program to assist. I saw a post on reddit announcing this application and was excited. Unfortunately, without the ability to select all the fonts included in the stock u8g2 library, this application is not useful at all for me. Selecting fonts in different sizes and making sure they match pixel wise is the biggest challenge. I can use any basic application like MS paint or GIMP to accomplish drawing lines, circles, boxes, etc.  

u8g2 includes the bdf files. If that is a challenge, to support the stock u8g2 stuff, perhaps using the 'c' code array and indexing into that might be easier?
